### ISSUE TITLE
Caret 4 Beta - Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by SteveEdson** [careteditor/caret#589]
The following error has occurred:
```
{
    "message": "Cannot read property 'length' of undefined",
    "version": "4.0.0-beta0",
    "os": "macOS High Sierra",
    "path": "extensions/Editor/Editor/Editor.js",
    "code": "(e.row-=1,e.column=this.lines[e.row].length),this.lines[e.row].visual",
    "line": 5,
    "column": 3302,
    "stack": [
        "at EditorDesktop.toRowColumn (extensions/Editor/Editor/Editor.js:5:3302)",
        "at EditorDesktop.removeLines (extensions/Editor/Editor/Editor.js:5:4509)",
        "at EditorMarkdownLines._wrapRange (extensions/Editor/Editor/Editor.Markdown.Lines.js:5:1242)",
        "at EditorMarkdownLines._onReWrapDocument (extensions/Editor/Editor/Editor.Markdown.Lines.js:5:951)",
        "at EditorDesktop.trigger (node_modules/desktop/Events.js:76:27)",
        "at EditorDesktop._resize (extensions/Editor/Editor/Editor.js:5:5132)"
    ]
}
```

**Comment by astoilkov** [comment]
Do you know how the bug can be replicated?

Thanks.

**Comment by SteveEdson** [comment]
I'm not sure sorry, I was playing around with Caret, opening tabs, looking at outlines, using the command palette and it randomly happened. Not been able to recreate it so far.

**Comment by astoilkov** [comment]
We think we fixed the problem in a new Caret 4.0 Beta update available [here](https://caret.io/4beta/).

Closing this issue. Let us know if this issue happens again.
